src/Libraries/PythonNodeModelsWpf/SharedCompletionProvider.cs

```diff
@@ -31,15 +32,22 @@ internal SharedCompletionProvider(PythonEngineVersion version ,string dynamoCore
             }
         }
 
-        internal static IExternalCodeCompletionProviderCore FindMatchingCodeCompletionCore 
+        internal static IExternalCodeCompletionProviderCore FindMatchingCodeCompletionCore
             (string versionName, ILogger logger = null)
         {
             try
             {
                 var completionType = typeof(IExternalCodeCompletionProviderCore);
                 var loadedCodeCompletionTypes = AppDomain.CurrentDomain.GetAssemblies()
-                    .SelectMany(s => s.GetTypes())
-                    .Where(p => completionType.IsAssignableFrom(p) && !p.IsInterface);
+                    .SelectMany(s => {
+                        try {
+                            return s.GetTypes();
+                        }
+                        catch (ReflectionTypeLoadException)
+                        {
+                            return new Type[0];
+                        }
+                    }).Where(p => completionType.IsAssignableFrom(p) && !p.IsInterface);
```

Comment:
One thing we could add is an explicit call to `ToList`. I see `loadedCodeCompletionTypes` is used afterwards a couple of times.

Reply:
sure will do - I tested it manually.